app/jobs/send_outgoing_text_message_job.rb

```diff
@@ -4,7 +4,7 @@ class SendOutgoingTextMessageJob < ApplicationJob
   def perform(outgoing_text_message_id)
     outgoing_text_message = OutgoingTextMessage.find(outgoing_text_message_id)
     begin
-      message = TwilioService.send_text_message(
+      message = TwilioService.new.send_text_message( # TODO: set explicit service scope; is this job for :ctc or :gyr?
```

Comment:
[DUST] again, would like to remove comments before merging

Reply:
absolutely. do you know if/how this job is aware of it's tenant service/environment?